acm/registration/odh-core.application.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: odh-core
+  namespace: rhoai-edge-acm
+spec:
+  componentKinds:
+  - group: apps.open-cluster-management.io
+    kind: Subscription
+  descriptor: {}
+  selector:
+    matchExpressions:
+      - key: app
+        operator: In
+        values: 
+          - odh-core
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ggithubcom-opendatahub-io-ai-edge-ns
```

Comment:
Very minor: is the extra `g` at the start here (and everywhere else that it's referenced) a typo, or was it intentional?